localstack/mock/infra.py

```diff
@@ -470,6 +480,49 @@ def dynamodb_extract_keys(item, table_name):
     return result
 
 
+# helper methods for response object
+def put_records_response(records, return_errors):
+    response = {
+        'FailedRecordCount': 0,
+        'Records': [
+        ]
+    }
+    for rec in records:
+        response = mock_record(response, return_errors)
+    return response
+
+
+def mock_record(response, return_errors):
+    if return_errors == 0:
+        record = good_record()
+    elif return_errors == 1:
+        record = unreliable_record()
+    else:
+        record = error_record()
+    if 'ErrorCode' in record:
+        response['FailedRecordCount'] += 1
+    response['Records'].append(record)
+    return response
+
+
+def good_record():
```

Comment:
I don't think we should be returning mock records here. In case of a "good" record, we still want to return the actual record returned by Kinesalite, right? In case of a "bad" record, we want to return `ProvisionedThroughputExceededException`.